src/engine/eip7547.md

```diff
@@ -0,0 +1,165 @@
+# Engine API -- EIP7547
+
+Engine API changes introduced in EIP7547.
+
+This specification is based on and extends [Engine API - Cancun](./cancun.md) specification.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Structures](#structures)
+  - [InclusionListEntryV1](#inclusionlistentryv1)
+  - [ExecutionPayloadV4](#executionpayloadv4)
+  - [PayloadAttributesV4](#payloadattributesv4)
+- [Methods](#methods)
+  - [engine_newPayloadV4](#engine_newpayloadv4)
+    - [Request](#request)
+    - [Response](#response)
+    - [Specification](#specification)
+  - [engine_forkchoiceUpdatedV4](#engine_forkchoiceupdatedv4)
+    - [Request](#request-1)
+    - [Response](#response-1)
+    - [Specification](#specification-1)
+  - [engine_getPayloadV4](#engine_getpayloadv4)
+    - [Request](#request-2)
+    - [Response](#response-2)
+    - [Specification](#specification-2)
+  - [engine_getInclusionListV1](#engine_getinclusionlistv1)
+    - [Request](#request-3)
+    - [Response](#response-3)
+    - [Specification](#specification-3)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Structures
+
+### InclusionListEntryV1
+
+The fields are encoded as follows:
+
+- `address`: `DATA` - 20 byte address of the entry.
+- `gasLimit`: `DATA` - 32 byte max amount of gas consumed by the transaction.
```

Comment:
should probably be `QUANTITY`?

Reply:
should not be bytes either :)

Reply:
fixed! 32 bytes would be a biiiiig amount of gas ;)